raiden/tests/utils/transfer.py

```diff
@@ -40,50 +42,34 @@ def get_channelstate(app0, app1, token_network_identifier) -> NettingChannelStat
     return channel_state
 
 
-def transfer(initiator_app, target_app, token, amount, identifier):
```

Comment:
This was previously used to do direct transfers. When the feature was removed the function was just repurposed to do mediated transfer, now this removes the duplicated code.